includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -381,8 +381,23 @@ protected function validate_fields( $values ) {
 						foreach ( $check_value as $file_url ) {
 							// Check image path.
 							$baseurl = wp_upload_dir()['baseurl'];
```

Comment:
This can be moved inside the `if( ! is_numeric( $file_url ) ) {` statement as it's only used specifically for that code (not a big deal but wanted to mention it)

Reply:
Ah sorry I forgot adding this change. Fixed in a6efd19ec4f31d14899c755ea08953796814f837